nebula-logger/core/main/log-management/classes/LogBatchPurger.cls

```diff
@@ -22,6 +41,17 @@ global with sharing class LogBatchPurger implements Database.Batchable<SObject>,
      */
     global LogBatchPurger() {
         this.totalProcessedRecords = 0;
+        this.pluginConfigurations = LoggerPlugin.getFilteredPluginConfigurations(
+            new List<Schema.SObjectField>{ Schema.LoggerPlugin__mdt.BatchPurgerApexClass__c, Schema.LoggerPlugin__mdt.BatchPurgerFlowName__c },
+            Schema.LoggerPlugin__mdt.BatchPurgerExecutionOrder__c
+        );
+
+        if (System.Test.isRunningTest() == true) {
```

Comment:
Hmmm, I hadn't thought of there being a use case for the getter methods to be `public`, but you're right that it could come in handy for downstream consumers. I also meant to remove `@TestVisible` on the maps - I started with just the maps, then later added the `@TestVisible` getter methods, but never removed it on the maps. I think for now, I'll...

- Remove `@TestVisible` on the maps
- Keep the current visibility on the 2 `@TestVisible` getter methods for this PR/release, but add a `TODO` for making them `public` in a future release. I think that seems like a great idea long-term, but I want to look into it a little more first before raising the visibility.

Reply:
Makes sense!